### DISCUSSION TITLE
PackageInfo: Error running npm outdated --json. Try running manually.

### DISCUSSION THREAD

**Discussion by winkee01** [vuki656/package-info.nvim#119] [Q&A]
I bind this key
```
vim.api.nvim_set_keymap(
    "n",
    "<leader>ns",
    "<cmd>lua require('package-info').show()<cr>",
    { silent = true, noremap = true }
)
```

then when I type `<leader>ns` on a package name, I get the error. Why?
<img width="657" alt="CleanShot 2022-04-29 at 22 01 43@2x" src="https://user-images.githubusercontent.com/49930785/165959682-71cb27d9-cc2d-4429-8583-1e5eb0653877.png">

I exit nvim and do a manually `npm outdated --json` on terminal, no problem.

**Comment by vuki656** [comment]
Hey,

can you try `ref/reorganize-code` branch and see if the problem persists?

**Reply by winkee01** [reply]
When I checked out `ref/reorganize-code` branch and re-press `<leader>ns`, I got this:
<img width="730" alt="CleanShot 2022-05-01 at 20 03 15@2x" src="https://user-images.githubusercontent.com/49930785/166145038-ef9a10c3-c800-4c08-8074-b074868446ae.png">

Also, I then checked out back to `master` branch, re-type `<leader>ns`, this time, it gave me this:
![image](https://user-images.githubusercontent.com/49930785/166145061-545e54ee-899c-4bf3-8024-6f9cfef7c358.png)

**Reply by vuki656** [reply]
Thats weird.

Can you post your `package.json` file?

Also, is the `package.json` file located at the root of the project?

Is it a monorepo or a monolith?

**Reply by winkee01** [reply]
Hi, I've tried different `package.json` in different projects, and all post the same errors, so I guess they are not about what `package.json` are.

**Reply by vuki656** [reply]
I'll sprinkle some logs within the code to see what's happening.

Probably within a day or two.
